index.html

```diff
@@ -10,25 +10,148 @@
+        </ul>
+        <div class="see">
+        <a href="#" class="text">See Project</a>
+        </div>
+    </div>
 
-    <section>
-        <h1>I'm Abhishek. <br> Glad to see <br> you !</h1>
-        <p>I'm a software developer! I can help you build a product, feature or website look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me. </p>
-        <a href="#">LET'S CONNECT</a>
-            <ul class="social">
+    <div class="box1">
+        <img src="/image/Snapshoot Portfolio (1).png">
+        <h1>Multi-Post <br>Stories
+        <ul>
+            <li class="point1">CANOPY</li>
+            <li class="point2">Back End Dev</li>
+            <li class="point3">2015</li>
+        </ul>
+    </h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="title">
+            <li class="lang1">HTML</li>
+            <li class="lang2">CSS</li>
+            <li class="lang3">JavaScript</li>
+        </ul>
+        <div class="see">
+        <a href="#" class="text">See Project</a>
+        </div>
+    </div>
+
+    <div class="box1">
+        <img src="/image/Snapshoot Portfolio (2).png">
+        <h1>Tonic
+        <ul>
+            <li class="point1">CANOPY</li>
+            <li class="point2">Back End Dev</li>
+            <li class="point3">2015</li>
+        </ul>
+    </h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="title">
+            <li class="lang1">HTML</li>
+            <li class="lang2">CSS</li>
+            <li class="lang3">JavaScript</li>
+        </ul>
+        <div class="see">
+        <a href="#" class="text">See Project</a>
+        </div>
+    </div>
+
+    <div class="box1">
+        <img src="/image/Snapshoot Portfolio (3).png">
+        <h1>Multi-Post<br>Stories
+        <ul>
+            <li class="point1">CANOPY</li>
+            <li class="point2">Back End Dev</li>
+            <li class="point3">2015</li>
+        </ul>
+    </h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="title">
+            <li class="lang1">HTML</li>
+            <li class="lang2">CSS</li>
+            <li class="lang3">JavaScript</li>
+        </ul>
+        <div class="see">
+        <a href="#" class="text">See Project</a>
+        </div>
+    </div>
+
+    <!-- About me Section -->
+    <div class="about">
+        <div class="base">
+            <div class="top">
+        <h1>About Myself</h1>
+        <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        </div>
+        <div class="social">
+            <a href="#">LET'S CONNECT</a>
```

Comment:
- [ ] You will agree with me that this isn't a link in itself although it introduces a list of links. So, kindly use an `hx` tag here other than `h1` since we can't have more than one in an entire document :pray: